lox_services/scraping/selenium_util.py

```diff
@@ -27,8 +28,7 @@ def wait_until_page_loaded(driver: webdriver.Chrome, timeout: int = 10):
     """
     try:
         WebDriverWait(driver, timeout).until(
-            lambda driver: driver.execute_script("return document.readyState")
-            == "complete"
+            lambda dr: dr.execute_script("return document.readyState") == "complete"
```

Comment:
please reverse this change

Reply:
Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes